From: Vipul <finn02@disroot.org>
Date: Wed, 19 Feb 2020 05:08:06 +0000
Subject: [Doc]: Fix typo in dict's man page

In dict's man page, "or" keyword is used twice times, instead of one.
---
 dict.1.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dict.1.in b/dict.1.in
index fce7d24..4385ded 100644
--- a/dict.1.in
+++ b/dict.1.in
@@ -52,7 +52,7 @@ section.
 .BI \-h " server\fR or " \-\-host " server"
 Specifies the hostname for the DICT server.  Server/port combinations can
 be specified in the configuration file.  If no servers are specified
-in the configuration file or or on the command line, 
+in the configuration file or on the command line,
 .B dict 
 will fail.  If IP lookup for a server expands to a list of IP addresses
 (as dict.org does currently), then each IP will be tried in the order
